numpy/_typing/_add_docstring.py

```diff
@@ -120,7 +120,7 @@ def _parse_docstrings() -> str:
 
 add_newdoc('NDArray', repr(NDArray),
     """
-    A `np.ndarray[Any, np.dtype[+ScalarType]] <numpy.ndarray>` type alias 
+    A `np.ndarray[Any, np.dtype[+ScalarType]] <numpy.ndarray>` type alias
```

Comment:
Another out-of-scope whitespace fix. 

(If you're using vscode, then I believe you can configure it to only strip trailing whitespaces for the lines that you actually edited, or something like that)